virt/virtual_machines/advanced_vm_management/virt-configuring-virtual-gpus.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-virtual-gpus"]
+= Configuring virtual GPUs
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-virtual-gpus
+
+toc::[]
+
+If you have graphics processing unit (GPU) cards, {VirtProductName} can automatically create virtual GPUs (vGPUs) that you can assign to virtual machines (VMs).
+
+include::modules/virt-about-using-virtual-gpus.adoc[leveloffset=+1]
+
+[id="preparing-hosts-mdevs_{context}"]
+== Preparing hosts for mediated devices
+
+You must enable the Input-Output Memory Management Unit (IOMMU) driver before you can configure mediated devices.
+
+include::modules/virt-adding-kernel-arguments-enable-iommu.adoc[leveloffset=+2]
+
+[id="configuring-nvidia-gpu-operator_{context}"]
+== Configuring the NVIDIA GPU Operator
+
+You can use the NVIDIA GPU Operator to provision worker nodes for running GPU-accelerated virtual machines (VMs) in {VirtProductName}.
+
+[NOTE]
+====
+The NVIDIA GPU Operator is supported only by NVIDIA. For more information, see link:https://access.redhat.com/solutions/5174941[Obtaining Support from NVIDIA] in the Red Hat Customer Portal knowledge base.
```

Comment:
from the [supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#knowledgebase):
```suggestion
The NVIDIA GPU Operator is supported only by NVIDIA. For more information, see link:https://access.redhat.com/solutions/5174941[Obtaining Support from NVIDIA] in the Red Hat Knowledgebase.
```

Reply:
Thank you! I knew that didn't sound quite right.